src/integration/utils/runTests/index.js

```diff
@@ -0,0 +1,97 @@
+/* eslint-disable no-console */
+
+const { exec, spawn } = require('child_process');
+const argv = require('minimist')(process.argv.slice(2));
+const ora = require('ora');
+const writeTestFiles = require('./writeTestFiles/writeTestFiles');
+const { SERVICES_TESTS_DIR } = require('./constants');
+
+const isCI = argv.ci;
+const isDev = Boolean(argv.dev);
+process.env.DEV_MODE = isDev;
+
+const getJestArgs = () =>
+  process.argv
+    .slice(2)
+    .filter(flag => !flag.startsWith('--services='))
+    .filter(flag => !flag.startsWith('--dev'));
+
+const getFilesToTest = services => {
+  if (services) {
+    const servicesRegexp = services.replace(/,/g, '|');
+    return `./src/integration/${SERVICES_TESTS_DIR}/(${servicesRegexp})/.+?`;
+  }
+  return `./src/integration/${SERVICES_TESTS_DIR}/.+?`;
+};
+
+const filesToTest = getFilesToTest(argv.services);
+
+const stopApp = () =>
+  new Promise(resolve => {
+    const child = exec('npm run stop');
+
+    child.on('exit', resolve);
+  });
+
+const buildApp = () =>
+  new Promise(resolve => {
+    const child = exec('npm run build:local');
+
+    child.on('exit', resolve);
+  });
+
+const startApp = () => {
+  return new Promise(resolve => {
+    const child = exec(
+      `npm run ${
+        isDev ? 'dev' : 'start'
+      } & ./node_modules/.bin/wait-on -t 20000 http://localhost:7080/status`,
+    );
+
+    child.on('exit', resolve);
+  });
+};
+
+const runTests = () =>
+  new Promise(resolve => {
+    const child = spawn(
+      'jest',
+      [filesToTest, '--runInBand', '--colors', ...getJestArgs()],
```

Comment:
I'm curious – what happens if we allow tests to run in parallel on local? 🤪

Reply:
We used to run them in parallel before right? Running in band seemed to avoid some of the tests timing out. Though I'm not sure why we never increased the timeout of those tests instead.